frontend/model/contracts/group.js

```diff
@@ -50,14 +50,15 @@ export function createInvite (
   responses: {...},
   status: string,
 |} {
+  const expiresInDate = creator === INVITE_INITIAL_CREATOR ? 30 : 7
```

Comment:
Since a number of days is not actually a date, I would rather name this constant e.g. `maxAgeInDays`

Reply:
I agree. Where should I place those constants.

Reply:
Is there any recommended file to store those constants?
What about this file `frontend/model/contracts/constants.js`?

Reply:
```
export const MAX_INVITE_AGE_IN_DAYS = {
  INITIAL: 30,
  PROPOSAL: 7
}
```

What about this in `frontend/model/contracts/constants.js`?

Reply:
In fact I was rather thinking about renaming the `expiresInDate` constant. Sorry I should have been more explicit about that.

Nonetheless, I think defining e.g. `MAX_INVITE_AGE_IN_DAYS `  in `constants.js` is a good idea; it improves consistency since that file already contains invite-related constants.